testproxy/services/read-rows.js

```diff
@@ -72,7 +72,13 @@ const readRows = ({clientMap}) =>
         rows: rows.map(getRowResponse),
       };
     } catch (e) {
-      return {status: e};
+      return {
+        status: {
+          code: e.code,
+          details: [],
+          message: e.message,
+        },
+      };
```

Comment:
The test proxy service was not sending back the status because the details weren't in an array format. This made it impossible for the test to check the code and message because status was nil.

Reply:
does `e` have details that could be surfaced? Or is details truly empty?

Reply:
It sounds like an underlying issue with the test runner that's not critical to this test - I made a suggestion to add a comment and open a buganizer bug that you then reference in the comment

Reply:
Sounds good